README.md

```diff
@@ -85,13 +85,16 @@ data class City(val name: String)
 Instances of these classes can now be passed [along with their serializer](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serializers.md#introduction-to-serializers) to the SDK:
 
 ```kotlin
-db.collection("cities").document("LA").set(City.serializer(), city, encodeDefaults = true)
+db.collection("cities").document("LA").set(City.serializer(), city) { shouldEncodeElementDefault = true }
```

Comment:
think we should keep the name encodeDefaults, as thats how all other formats surface this property to clients
<img width="1026" alt="image" src="https://github.com/GitLiveApp/firebase-kotlin-sdk/assets/1386607/b3cf1ec9-1563-462f-8747-162bbc0535fa">

Reply:
Renamed